ProjectSidewalk/SidewalkWebpage
# #2804:  Duplicate images moved to be referenced by config file

Resolves #2804

Removed duplicate sidewalk logo and skyline images and moved them to the public/assets folder, assigned each city to a specific skyline image in the cityparams.conf file, and adjusted the navbar and index page to use the images from their new path locations.

##### Before/After screenshots (if applicable)

Should look exactly the same before and after!

##### Testing instructions
1. Open the live version of Project Sidewalk in one tab and the test server in another tab.
2. Check the Project Sidewalk logo in the top left of the navbar and the skyline background image (scroll down to the "Your Work is Making a Difference!" and stats part of the page).
3. If both look the same in the live and test version, then it works!
4. Repeat for all city servers

##### Things to check before submitting the PR
